python/tank/deploy/tank_commands/app_info.py

```diff
@@ -30,13 +30,20 @@ def __init__(self):
 
     def run_noninteractive(self, log, parameters):
         """
-        API accessor
+        Tank command API accessor. 
+        Called when someone runs a tank command through the core API.
+        
+        :param log: std python logger
+        :param parameters: dictionary with tank command parameters
```

Comment:
`run_noninteractive` is always called from the command API, it's just implementing a base class method, that's why it's got the parameter. I'll add a clarifying comment.